crates/external_services/src/grpc_client.rs

```diff
@@ -0,0 +1,238 @@
+use std::fmt::Debug;
+
+pub use common_utils::errors::CustomResult;
+use common_utils::{ext_traits::OptionExt, id_type, transformers::ForeignFrom};
+pub use dynamic_routing::{
+    success_rate_calculator_client::SuccessRateCalculatorClient, CalSuccessRateConfig,
+    CalSuccessRateRequest, CalSuccessRateResponse,
+    CurrentBlockThreshold as DynamicCurrentThreshold, LabelWithStatus,
+    UpdateSuccessRateWindowConfig, UpdateSuccessRateWindowRequest, UpdateSuccessRateWindowResponse,
+};
+use error_stack::ResultExt;
+use hyperswitch_interfaces::api::api_models::routing::{
+    CurrentBlockThreshold, DynamicRoutingConfig, DynamicRoutingConfigBody, RoutableConnectorChoice,
+    RoutableConnectorChoiceWithStatus,
+};
+use router_env::logger;
+use tonic::transport::Channel;
+#[allow(
```

Comment:
I don't feel right if we evade `clippy` this way, what do you guys think?

cc: @SanchithHegde @Chethan-rao